activerecord/test/cases/base_test.rb

```diff
@@ -1779,22 +1778,12 @@ def test_protected_environments_are_stored_as_an_array_of_string
 
   test "#connected_to_many sets prevent_writes if role is reading" do
     ActiveRecord::Base.connected_to_many([SecondAbstractClass], role: :reading) do
-      assert SecondAbstractClass.current_preventing_writes
```

Comment:
Instead of deleting these we should update the assertions.

Reply:
Thanks @eileencodes! updated